Chat_App/Extensions/Constants.swift

```diff
@@ -0,0 +1,81 @@
+//
+//  File.swift
+//  Chat_App
+//
+//  Created by Ana Dzamelashvili on 5/4/23.
+//
+
+import Foundation
+import UIKit
+
+enum Constants {
```

Comment:
You can create a private extension for your views and keep all constants there. For instance:
`private extension TextView {
       enum Constants { ... }
} `
`private extension SendButton {
       enum Constants { ... }
} `

Reply:
you mean to create separate private extension files for every view?

Reply:
yes